api/test/unit/utils/test_Pagination.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from catalog.api.utils.pagination import StandardPagination
+from django.core.paginator import Paginator as DjangoPaginator
+from django.db import models
+from django.test import TestCase
+ 
+from rest_framework import (
+   exceptions, filters, generics, pagination, serializers, status
+)
+from rest_framework.pagination import PAGE_BREAK, PageLink
+from rest_framework.request import Request
+from rest_framework.test import APIRequestFactory
+ 
+factory = APIRequestFactory()
+ 
+class TestPagination:
+ 
+ def setup_method(self):
+       class PassThroughSerializer(serializers.BaseSerializer):
+           def to_representation(self, item):
+               return item
+ 
+       class EvenItemsOnly(filters.BaseFilterBackend):
+           def filter_queryset(self, request, queryset, view):
+               return [item for item in queryset if item % 2 == 0]
+ 
+       class BasicPagination(pagination.PageNumberPagination):
```

Comment:
The goal of the issue #401 was to test our `StandardPagination` class.

https://github.com/WordPress/openverse-api/blob/e3a5cfb9e7a88f8afb8167b6e6072fa5115e1e53/api/catalog/api/utils/pagination.py#L6

You should change the pagination class in your view to use `StandardPagination`.